server/app/assets/javascripts/mapquestion/map_question_filters.ts

```diff
@@ -90,18 +92,30 @@ const applyLocationFilters = (
       }
     }
   })
-  updateLocationCountForMap(mapId)
+
+  updateLocationCountForMap(
+    mapId,
+    visibleCount,
+    locationCheckboxContainers.length,
```

Comment:
Part of the nitpick:

```suggestion
    totalCount,
```